CHANGELOG.md

```diff
@@ -115,83 +137,86 @@ The following patterns are _not_ considered violations:
+- Replaced deprecated `scss/at-mixin-no-argumentless-call-parentheses` rule with its equivalent `scss/at-mixin-argumentless-call-parentheses`
+- `eslint-plugin-shopify` to the latest version, and updated ESLint to the appropriate version
 
 ### Changed
-* `media-feature-name-no-unknown` to ignore `prefers-reduced-motion`
+
+- `media-feature-name-no-unknown` to ignore `prefers-reduced-motion`
 
 ## [2.0.1] - 2017-07-28
 
 ### Changed
-* Set `selector-max-type` to 1
+
+- Set `selector-max-type` to 1
 
 ## [2.0.0] - 2017-07-27
 
 ### Added
 
-* New plugin:
-  * Added `stylelint-order` which replaces `declaration-block-properties-order`
-
-* New rules:
-  * `rule-empty-line-before`
-  * `selector-max-universal`
-  * `at-rule-semicolon-space-before`
-  * `selector-max-attribute`
-  * `selector-max-class`
-  * `selector-max-combinators`
-  * `selector-max-id`
-  * `selector-max-type`
-  * `function-url-scheme-blacklist` (disabled)
-  * `media-feature-name-whitelist` (disabled)
-  * `time-min-milliseconds` (disabled)
+- New plugin:
 
-### Removed
+  - Added `stylelint-order` which replaces `declaration-block-properties-order`
 
-* Deprecated rules:
-  * `block-no-single-line`
-  * `no-indistinguishable-colors`
-  * `declaration-block-no-ignored-properties`
-  * `declaration-block-properties-order`
-  * `function-url-data-uris`
-  * `no-browser-hacks`
-  * `no-unsupported-browser-features`
-  * `media-feature-no-missing-punctuation`
-  * `custom-property-no-outside-root`
-  * `root-no-standard-properties`
-  * `rule-nested-empty-line-before`
-  * `rule-non-nested-empty-line-before`
+- New rules:
+  - `rule-empty-line-before`
+  - `selector-max-universal`
+  - `at-rule-semicolon-space-before`
+  - `selector-max-attribute`
+  - `selector-max-class`
+  - `selector-max-combinators`
+  - `selector-max-id`
+  - `selector-max-type`
+  - `function-url-scheme-blacklist` (disabled)
+  - `media-feature-name-whitelist` (disabled)
+  - `time-min-milliseconds` (disabled)
 
+### Removed
+
+- Deprecated rules:
+  - `block-no-single-line`
+  - `no-indistinguishable-colors`
+  - `declaration-block-no-ignored-properties`
+  - `declaration-block-properties-order`
+  - `function-url-data-uris`
+  - `no-browser-hacks`
+  - `no-unsupported-browser-features`
+  - `media-feature-no-missing-punctuation`
+  - `custom-property-no-outside-root`
+  - `root-no-standard-properties`
+  - `rule-nested-empty-line-before`
+  - `rule-non-nested-empty-line-before`
 
 ### Changed
 
-* Properties order for shorthand notation with margin, padding, border styles have been updated to follow:
+- Properties order for shorthand notation with margin, padding, border styles have been updated to follow:
+
 ```
 property: <top> <right> <bottom> <left>
 ```
 
 ## 1.0.0 - 2017-05-29
-* Initial release
 
+- Initial release
 
-[Unreleased]: https://github.com/Shopify/stylelint-config-shopify/compare/v5.1.1...HEAD
+[unreleased]: https://github.com/Shopify/stylelint-config-shopify/compare/v5.1.1...HEAD
```

Comment:
"Unreleased" should stay capitalized.

Also, it looks like some diff links are missing for versions 5.1.1 to 6.1.0.